### PR TITLE
Better read_x_values and fix file order in _load_images

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: ess/v20/imaging/helper_funcs.py [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
+                      usecols=usecols)
+    if (usecols is not None) or (data.ndim == 1):
+        return data
+
+    # Search for the first column with monotonically increasing values
+    for i in range(data.shape[1]):
+        if np.all(data[1:, i] > data[:-1, i], axis=0):
+            return data[:, i]
+
+    raise RuntimeError("No column with monotonically increasing values was "
```
Comment (OwenArnold) [inline ess/v20/imaging/helper_funcs.py]: Extend message "Cannot automatically determine time-of-flight column."

File: ess/v20/imaging/helper_funcs.py [hunk 2]
```diff
@@ -1,30 +1,46 @@
...
+import re
 
 import numpy as np
 import scipp as sc
 import tifffile
 from astropy.io import fits
 
 
-def read_x_values(tof_file):
+def read_x_values(tof_file, delimiter=None, skiprows=0, usecols=None):
```
Comment (OwenArnold) [inline ess/v20/imaging/helper_funcs.py]: minor point - worth adding params for these new inputs

**Review by OwenArnold — APPROVED** [review]
Some additions suggested, but these can be done in follow up as this is probably time critical.

File: ess/v20/imaging/helper_funcs.py (hunk 2 above)
Comment (nvaytet) [reply ess/v20/imaging/helper_funcs.py]: you mean in the docstring?

Comment (OwenArnold) [reply ess/v20/imaging/helper_funcs.py]: Yes

**Pull request merged**